src/components/Proposals/Proposals.js

```diff
@@ -1,15 +1,36 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
+import { CardLayout, GU, useLayout } from '@aragon/ui'
+import { createAppHook, useApp } from '@aragon/connect-react'
+import connectVoting from '@aragon/connect-voting-disputable'
+import ProposalCard from './ProposalCard'
+
+const useDisputableVoting = createAppHook(connectVoting)
 
 function Proposals() {
+  const [disputableVoting] = useApp('disputable-voting')
+  const [votes] = useDisputableVoting(disputableVoting, (app) => app.votes())
+  const { layoutName } = useLayout()
+  const compactMode = layoutName === 'small'
+  const rowHeight = compactMode ? null : 294
```

Comment:
How do you feel about setting this to always be `294` for now and removing `useLayout`? The layout context doesn't exist yet so I think this would be equivalent anyway. We can come back and tweak it after!